### DISCUSSION TITLE
how to enable json post

### DISCUSSION THREAD

**Discussion by gavenwan** [fastapi-users/fastapi-users#573] [Q&A] [answered]
as referred to documents,
we should do post use **FormData** like below :
```
const formData = new FormData();
formData.set('username', 'king.arthur@camelot.bt');
formData.set('password', 'guinevere');
axios.post(
    'http://localhost:8000/auth/jwt/login',
    formData,
    {
        headers: {
            'Content-Type': 'multipart/form-data',
        },
    },
)
.then((response) => console.log(response))
.catch((error) => console.log(error));
```
However,for some reason,my frontwebpage(Vue-admin) use json to post data like this
`
axios.post(url, data).then().catch().
`
post json data is not allowed in fastapi-user ,what Can I do to enable json post(not just fomdata post)

**Comment by frankie567** [comment, marked as answer]
Hi there!

Form data is required only for the `/login` endpoint (because of the OAuth specification). The other endpoints expect JSON, no problem about that.

I see you also use `axios` in your VueJS application, so why you can't use the code shown in example just for login?
